fennel/client/client.py

```diff
@@ -311,6 +306,8 @@ def init_branch(self, name: str):
         Raises an exception if a branch with the same name already exists.
 
         """
+        print(f"Creating branch {name}...")
```

Comment:
Stray?

Reply:
I felt its useful, to know,  since sometimes you can have invalid names etc, now its part of the log

Reply:
Upto you. In my view, we haven't put these prints anywhere else. And I think the real issue is that print is not a reliable communication mechanism. If there was any information that we wanted user to know, it should either be in return value or exception.